charts/aad-pod-identity/templates/crd.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Any specific reason why this is named in singular - crd instead of crds (like identities file ?)

Reply:
No reason :) we can change it.